packages/squiggle-lang/src/fr/list.ts

```diff
@@ -687,6 +694,19 @@ List.reduceWhile([5, 6, 7], { x: 0 }, {|acc, curr| { x: acc.x + curr }}, {|acc|
       makeDefinition([frArray(frString)], frString, ([array]) => array.join()),
     ],
   }),
+  maker.make({
+    name: "concat",
```

Comment:
Is this a duplicate for another `concat` definition above?

Reply:
There are two concat definitions now - one for strings, that only works when the first element is a string. The second is here, which works for lists of any item.

Reply:
There's another `concat` here, in this file, that's defined on arrays (with slightly better generic types).

Here: https://github.com/quantified-uncertainty/squiggle/blob/0ffbd187a5af404428dbe5a2a8a3573fae50ab9b/packages/squiggle-lang/src/fr/list.ts#L274

Reply:
Oh, great point, will remove this then. I guess this was there before accidentally.